setup.py

```diff
@@ -811,11 +812,6 @@ def is_package_excluded(package: str, exclusion_list: List[str]):
     return any(package.startswith(excluded_package) for excluded_package in exclusion_list)
 
 
-devel_all = [
-    package
-    for package in devel_all
-    if not is_package_excluded(package=package, exclusion_list=PACKAGES_EXCLUDED_FOR_ALL)
-]
```

Comment:
If we remove the "snakebite if" then `PACKAGES_EXCLUDED_FOR_ALL` is empty so this line do no changes, so we should probably remove it?